src/main/java/com/powsybl/openloadflow/network/LfContingency.java

```diff
@@ -149,7 +149,7 @@ public void apply(LoadFlowParameters.BalanceType balanceType) {
             PowerShift shift = lostLoad.getPowerShift();
             load.setTargetP(load.getTargetP() - getUpdatedLoadP0(load, balanceType, shift.getActive(), shift.getVariableActive()));
             load.setTargetQ(load.getTargetQ() - shift.getReactive());
-            load.setAbsVariableTargetP(load.getAbsVariableTargetP() - Math.abs(shift.getVariableActive()));
+            load.setAbsVariableTargetP(load.getAbsVariableTargetP() - getUpdatedLoadAbsVariableTargetP(load, shift.getVariableActive()));
             lostLoad.getOriginalIds().forEach(loadId -> load.setOriginalLoadDisabled(loadId, true));
```

Comment:
Why do you need to do somthing special for fictive load in setAbsVariableTargetP for firktive loads since the getAbsVariableTargetP is harcoded for fiktive load , by this new code::
   private double getAbsVariableTargetP(Load load) {
        if (isLoadFictitious(load)) {
            return 0.0;
        }

Reply:
Ha, you are right.
I didn't realize that.
Reverted back to what it was.